docs/dev/protocol-extensions.md

```diff
@@ -197,16 +197,14 @@ use that information for every subsequent query.
 If we send the query to the wrong node/shard, we want to send the RESULT 
 message with additional information about the tablet in `custom_payload`:
 
-  - `tablet_replicas` - information about tablet replicas, for every
-    replica there is information about the host and shard.
-  - `token_range` - information about token range for that tablet in
-    format `(first_token, last_token]`.
+  - `tablets-routing-v1` - tablets routing information, which contains info about token
+    range (in format `(first_token, last_token]`) and tablet replicas, for every replica
+    there is information about the host and shard.
 
-The driver has to be able to receive `custom_payload` and deserialise its fields
+The driver has to be able to receive `custom_payload` and deserialise its field
 from `bytes` to:
 
-  - for `tablet_replicas` - `ListType(TupleType(UUIDType, Int32Type))`.
-  - for `token_range` - two `LongType`.
+  - for `tablets-routing-v1` - ` [long][long] [[short][short][uuid][short][int] list]`.
```

Comment:
I'm missing the description of what each of these `[long]` `[short]` etc. represents.

Reply:
Yes, it's also better to give a name to the list element, like in the protocol spec.

Reply:
is it better now?